ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -430,6 +430,27 @@ export async function submitMhrTransfer (payloadData, mhrNumber, staffPayment) {
   }
 }
 
+// Register a Unit Note on an existing manufactured home.
+export async function submitMhrUnitNote (mhrNumber, payloadData) {
+  try {
+    const result = await axios.post(`notes/${mhrNumber}`, payloadData, getDefaultConfig())
+    if (!result?.data) {
+      throw new Error('Invalid API response')
+    }
+    return result.data
+  } catch (error: any) {
+    return {
+      error: {
```

Comment:
The default error information is there in case the payload is incorrect and API throws an error vs returning a proper error response. cc: @doug-lovett

Reply:
Good call: the response could come from the gateway and the request may never reach the registries internal API.